cmd/vclusterctl/cmd/platform/destroy.go

```diff
@@ -0,0 +1,105 @@
+package platform
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/loft-sh/log"
+	"github.com/loft-sh/vcluster/pkg/cli/destroy"
+	"github.com/loft-sh/vcluster/pkg/cli/flags"
+	"github.com/loft-sh/vcluster/pkg/cli/start"
+	"github.com/loft-sh/vcluster/pkg/platform/clihelper"
+	"github.com/spf13/cobra"
+)
+
+type DestroyCmd struct {
+	destroy.DeleteOptions
+}
+
+func NewDestroyCmd(globalFlags *flags.GlobalFlags) *cobra.Command {
+	cmd := &DestroyCmd{
+		DeleteOptions: destroy.DeleteOptions{
+			Options: start.Options{
+				GlobalFlags: globalFlags,
+				Log:         log.GetInstance(),
+				CommandName: "destroy",
+			},
+		},
+	}
+
+	destroyCmd := &cobra.Command{
+		Use:   "destroy",
+		Short: "Destroy a vCluster Platform instance",
+		Long: `########################################################
+############# vCluster Platform destroy ##################
```

Comment:
I think this one can stay lower-cased, as it is the CLI commands as a banner here.

Reply:
sharp eye, thank you!